test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +21,59 @@ fi
 
 success=0
 failure=0
+running=0
+# Jobs to execute in parallel (hard coded to 8)
+parallel_jobs=8
+PID=""
+declare -a PID_LIST
 
+# Wait for other format runs.
+wait_for_other_format_runs()
+{
+	for process in ${PID_LIST[@]};do
+		# Remove the process id of the completed runs from the process id array.
+		unset PID_LIST[0]
+		PID_LIST=(${PID_LIST[@]})
+		wait $process
+		exit_status=$?
+		let "running--"
+		config_name=`egrep $process $tmp_file | awk -F ":" '{print $2}' | rev | awk -F "/" '{print $1}' | rev`
+		if [ $exit_status -ne "0" ]; then
+			let "failure++"
+			[ -f WT_TEST_${config_name}/CONFIG ] && cat WT_TEST_${config_name}/CONFIG
+		else
+			let "success++"
+			# Remove database files of successful jobs.
+			[ -d WT_TEST_${config_name} ] && rm -rf WT_TEST_${config_name}
+		fi
+
+		return
+	done
+}
+
+tmp_file="format_list.txt"
+touch $tmp_file
 # Cycle through format CONFIGs recorded under the "failure_configs" directory
 for config in $(find ../../../test/format/failure_configs/ -name "CONFIG.*" | sort)
 do
 	echo -e "\nTesting CONFIG $config ...\n"
-	if (./t -1 -c $config -h WT_TEST); then
-		let "success++"
-	else
-		let "failure++"
-		[ -f WT_TEST/CONFIG ] && cat WT_TEST/CONFIG
-		mv WT_TEST WT_TEST_$(basename $config)
+	basename_config=$(basename $config)
```

Comment:
Do any of the fail configs require abort and restart? I'm assuming not and it might be worth adding a comment in there about it.

Reply:
Fixed in 913a1ba0e1fca8768e30cd529b12f8703ab90b27

Reply:
Do any of the fail configs require abort and restart?

Reply:
> Do any of the fail configs require abort and restart?

No, none of the configs require abort and restart.